### PR TITLE
Add functionality for removing task by click

### PR DESCRIPTION
**Pull request by XeniaVi**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,17 +27,30 @@ class Task {
...
+        tasks = res;
+        e.target.parentNode.remove();
+    }
 }
 
 const addTask = () => {
     const value = addInput.value;
     if (value !== '') {
-        const newTask = new Task(value);
+        let id = Number(new Date);
```
Comment (nortoza) [inline index.js]: If you wanna to get timestamp you can use Date.now(). And lets make this variable as const

File: index.js [hunk 2]
```diff
@@ -26,17 +27,30 @@ class Task {
         item.append(deleteBtn);
         taskInner.append(item);
 
+        deleteBtn.addEventListener('click', this.deleteTask)
+
         addInput.value = '';
     }
+
+    deleteTask(e) {
```
Comment (igirin) [inline index.js:35]: Lets move that function outside the class

File: index.js [hunk 3]
```diff
@@ -26,17 +27,30 @@ class Task {
...
+        const res = tasks.filter(item => {
+            return item.id !== this.id;
```
Comment (nortoza) [inline index.js]: We can remove {} brackets and return to write it in one line

File: index.js [hunk 4]
```diff
@@ -1,10 +1,11 @@
 const addInput = document.getElementById('add-input');
```
Comment (nortoza) [inline index.js]: Do we really need to make this variables are global?

File: index.js [hunk 5]
```diff
@@ -26,17 +27,30 @@ class Task {
...
+            return item.id !== this.id;
+        })
+        tasks = res;
+        e.target.parentNode.remove();
+    }
 }
 
 const addTask = () => {
     const value = addInput.value;
     if (value !== '') {
```
Comment (nortoza) [inline index.js]: We can replace it by "if (value)"

**Pull request merged**